.circleci/config.yml

```diff
@@ -296,29 +304,29 @@ jobs:
     macos:
       xcode: "11.4.0"
     steps:
-      - install-rust
-      - setup-rust-toolchain
-      - checkout
-      - setup-sccache
-      - build-ios-libs
       # We do not use the ssccache cache helper commands as
       # the macOS cache is in a different folder.
       - restore_cache:
           name: Restore sccache cache
           key: sccache-cache-macos-{{ arch }}
+      - checkout
+      - install-rust
+      - setup-rust-toolchain
       - run:
-          name: Setup build environment
+          name: Toggle brew auto-updates
           command: |
-            rustup target add aarch64-apple-ios x86_64-apple-ios
             if [ -z "${CIRCLE_TAG}" ]; then
               # On non-release builds do not update brew (takes ages).
               echo 'export HOMEBREW_NO_AUTO_UPDATE=1' >> $BASH_ENV
             fi
-            brew outdated carthage || brew upgrade carthage
-            carthage bootstrap --platform iOS
+      - setup-sccache
+      - build-ios-libs
+      - run:
+          name: Set up the build environment
+          command: |
+            rustup target add aarch64-apple-ios x86_64-apple-ios
             brew install swift-protobuf
-            # See https://circleci.com/docs/2.0/testing-ios/#pre-starting-the-simulator
-            xcrun instruments -w "iPhone 8 (13.0) [" || true
```

Comment:
This line didn't do anything (13.0 simulators are not present on newer XCode images). Also I've benchmarked with and without pre-starting a simulator and I get the same results.